### PR TITLE
pulsar-post-tasks as a role

### PR DESCRIPTION
**Pull request by cat-bro**
change the perms from 750 to 754 on pulsar roles so that regular users can read files in working dirs.  Group these into a role because all pulsars need the same post tasks

### CODE REVIEW

File: roles/pulsar-post-tasks/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+- name: Install slurm-drmaa
+  package:
+    name: slurm-drmaa1
+    state: present
+- name: own the /mnt/pulsar dirs as ubuntu
+  file:
+      path: /mnt/pulsar
```
Comment (Slugger70) [inline roles/pulsar-post-tasks/tasks/main.yml:8]: Change these to the variable for pulsar root...

File: roles/pulsar-post-tasks/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      owner: ubuntu
+      group: ubuntu
+      state: directory
+      recurse: yes
+      mode: 0754
+- name: own the /mnt/galaxy dirs as ubuntu
```
Comment (Slugger70) [inline roles/pulsar-post-tasks/tasks/main.yml:14]: same here

**Review by Slugger70 — CHANGES_REQUESTED** [review]

File: roles/pulsar-post-tasks/tasks/main.yml (hunk 1 above)
Comment (cat-bro) [reply roles/pulsar-post-tasks/tasks/main.yml]: good thinking

File: roles/pulsar-post-tasks/tasks/main.yml (hunk 2 above)
Comment (cat-bro) [reply roles/pulsar-post-tasks/tasks/main.yml]: I've added this as `galaxy_dir` in pulsarservers.yml.  I didn't want it to clash with galaxy_root from group_vars/galaxyservers.yml even though it is not in scope here

**Comment by cat-bro** [comment]
I put perms up to 755 for /mnt/pulsar so users can cd.  Is this getting too permissive?  An alternative would be adding ubuntu to the devs group in the common role, then the machine users would have r and x.

**Review by Slugger70 — APPROVED** [review]

**Pull request merged**